presto-native-execution/src/test/java/com/facebook/presto/spark/PrestoSparkNativeQueryRunnerUtils.java

```diff
@@ -103,16 +104,29 @@ public static Map<String, String> getNativeExecutionSessionConfigs()
 
     public static PrestoSparkQueryRunner createPrestoSparkNativeQueryRunner()
     {
-        PrestoSparkQueryRunner queryRunner = createPrestoSparkNativeQueryRunner(
-                Optional.of(getBaseDataPath()),
-                getNativeExecutionSessionConfigs(),
-                getNativeExecutionShuffleConfigs(),
-                ImmutableList.of(new NativeExecutionModule()));
+        PrestoSparkQueryRunner queryRunner = createPrestoSparkNativeQueryRunner(new NativeExecutionModule(Optional.empty()));
         setupJsonFunctionNamespaceManager(queryRunner);
 
+        return queryRunner;
+    }
+
+    private static PrestoSparkQueryRunner createPrestoSparkNativeQueryRunner(NativeExecutionModule nativeExecutionModule)
+    {
         // Increases log level to reduce log spamming while running test.
         customizeLogging();
-        return queryRunner;
+        return createPrestoSparkNativeQueryRunner(
+                Optional.of(getBaseDataPath()),
+                getNativeExecutionSessionConfigs(),
+                getNativeExecutionShuffleConfigs(),
+                ImmutableList.of(nativeExecutionModule));
+    }
+
+    // Similar to createPrestoSparkNativeQueryRunner, but with custom connector config and without jsonFunctionNamespaceManager
+    public static PrestoSparkQueryRunner createPrestoSparkNativeTPCHQueryRunner()
```

Comment:
These names are long and hard to read. Since PrestoSparkNativeQueryRunner is part of the class name, these methods can be shortened to:

createHiveRunner and createTpchRunner